lib/utilities/global-npm-version.js

```diff
@@ -0,0 +1,9 @@
+// Get the global npm version. You cannot require global npm modules, including
+// global npm itself, but you can still *run* global npm.
+const spawnSync = require('child_process').spawnSync;
```

Comment:
Noted on all fronts (including `const` and `spawnSync`—I didn't realize Ember CLI supported Node 0.10!); I'll push up an alternate commit with the no-npm-version-at-all strategy and rename the PR accordingly in a little bit.